docs/docs/integrations/language-models/volcengine-ark.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 2
+---
+
+# Ark
+
+- [Volcengine Ark Documentation](https://www.volcengine.com/docs/82379)
+- [Volcengine Ark Reference](https://www.volcengine.com/docs/82379/1263482)
+
+## Maven Dependency
+
+```xml
+<dependency>
+    <groupId>dev.langchain4j</groupId>
+    <artifactId>langchain4j-ark</artifactId>
+    <version>0.32.0</version>
```

Comment:
Suggestion: Change to project version  ${project.parent.version}

Reply:
change to 0.33.0